public/css/style.css

```diff
@@ -154,3 +154,13 @@ header {
     padding: 15px 45px;
   }
 }
+
+/*Assessment Styling*/
+.container{
+  justify-content: center;
+  text-align: center;
+}
```

Comment:
You shouldn't do this, this is a class styling from bootstrap which is used in other pages of the application so if you do this you're also changing css in other pages that use this class. Just add another class to where you want this css and change like 159 to that class.